Indicators/WilderSwingIndex.cs

```diff
@@ -0,0 +1,286 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Linq;
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// This indicator calculates the Swing Index (SI) as defined by Welles Wilder
+    /// in his book 'New Concepts in Technical Trading Systems'.
+    /// <para>
+    /// SIₜ = 50 * ( N / R ) * ( K / T )
+    /// </para>
+    /// <para>
+    ///   Where:
+    ///   <list type="bullet">
+    ///     <item>
+    ///       <term>N</term>
+    ///       <description>
+    ///         Equals: Cₜ - Cₜ₋₁ + 0.5 * (Cₜ - Oₜ) + 0.25 * (Cₜ₋₁ - Oₜ₋₁)
+    ///         <para>
+    ///           <i>See <see cref="GetNValue"/></i>
+    ///         </para>
+    ///       </description>
+    ///     </item>
+    ///     <item>
+    ///       <term>R</term>
+    ///       <description>
+    ///         Found by selecting the expression with the largest value and
+    ///         then using the corresponding formula.
+    ///         <para>
+    ///           <i>Expression</i> ⭢ <i>Formula</i>
+    ///           <list type="number">
+    ///             <item>
+    ///               <description>
+    ///                 |Hₜ - Cₜ₋₁|
+    ///                 ⭢
+    ///                 |Hₜ - Cₜ - 0.5 * (Lₜ - Cₜ₋₁) + 0.25 * (Cₜ₋₁ - Oₜ₋₁)|
```

Comment:
Does this pass XML spec; I'm not sure it'll accept special characters

Reply:
Fixed in 0f22f76.